### PR TITLE
fix: reschedule timezone

### PR DESCRIPTION
**Pull request by Udit-takkar**
https://threads.com/34524526047?s=iBF5JA9HyxtgCPWMv4fuRW

### CODE REVIEW

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -643,6 +643,7 @@ async function handler(
     notes: additionalNotes,
     smsReminderNumber,
     rescheduleReason,
+    bookingAttendees,
```
Comment (CarinaWolli) [inline packages/features/bookings/lib/handleNewBooking.ts:646]: Can't we just get the attendees from the `orginalRescheduleBooking`?

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 2]
```diff
@@ -848,6 +847,46 @@ async function handler(
...
+      !!eventType.seatsPerTimeSlot
+    );
+    if (!originalRescheduledBooking) {
+      throw new HttpError({ statusCode: 404, message: "Could not find original booking" });
+    }
+  }
+
+  const isOrganizerRescheduling = organizerUser.id === userId;
+
+  const attendeeInfoOnReschedule =
```
Comment (CarinaWolli) [inline packages/features/bookings/lib/handleNewBooking.ts:880]: only if the organizer reschedules we want the orignal tz and language. Attendee might be in different tz now and that's why he rescheduled with different tz

Comment (CarinaWolli) [reply packages/features/bookings/lib/handleNewBooking.ts:880]: @emrysal what do you think about that? 

@Udit-takkar's thought: i thought of this edge but if organiser clicks on reschedule from email or just open the reschedule on another device where he is not logged in then we won't be able to know if it is organiser or attendee who is rescheduling

**Review by emrysal — APPROVED** [review]
lgtm 🚀

**Review by shivamklr — COMMENTED** [review]
Found a minor [issue](https://github.com/calcom/cal.com/issues/10735).

**Pull request merged**